packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -732,10 +732,13 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
       case TargetPlatform.windows:
         break;
     }
+    final ShapeBorder? shape = expansionTileTheme.shape ?? widget.shape;
+    final ShapeBorder? collapsedShape = expansionTileTheme.collapsedShape ?? widget.collapsedShape;
+    final ShapeBorder? lerpShape = ShapeBorder.lerp(collapsedShape, shape, _animationController.value);
```

Comment:
```suggestion
```

I just realized—the `_border` getter is already evaluating the animation for us, so we can just use the existing variable:

```dart
    final ShapeBorder expansionTileBorder = _border.value ?? const Border(
      top: BorderSide(color: Colors.transparent),
      bottom: BorderSide(color: Colors.transparent),
    );
```